resources/js/components/RelationshipFormItem.vue

```diff
@@ -117,17 +117,18 @@
             },
 
             fill(formData, parentAttrib) {
+	            formData.append(`${parentAttrib}[${this.id}][modelId]`, this.modelId);
```

Comment:
Needs some formatting
```suggestion
	          formData.append(`${parentAttrib}[${this.id}][modelId]`, this.modelId);
	          
```